### PR TITLE
docs: fix some logger plugin metadata description

### LINKED ISSUE #6657: docs: clickhouse-logger-plugin&file-logger-plugin metadata set description error
### Current State

https://github.com/apache/apisix/blob/c7755543ae2e097942da4bbc5a9c6136de90fd78/docs/zh/latest/plugins/file-logger.md?plain=1#L73

https://github.com/apache/apisix/blob/c7755543ae2e097942da4bbc5a9c6136de90fd78/docs/zh/latest/plugins/clickhouse-logger.md?plain=1#L84

also kafka-logger-plugin zh/doc

at en docs,lack of this part of  description

### Desired State

for file-logger : "将对所有绑定 http-logger 的 Route 或 Service 生效。" -> "将对所有绑定 file-logger 的 Route 或 Service 生效。"

for clickhouse-logger : "将对所有绑定 http-logger 的 Route 或 Service 生效。" -> "将对所有绑定 clickhouse-logger 的 Route 或 Service 生效。"

I can fix.

### PR DESCRIPTION
**Pull request by liangliang4ward**
### Description

fix kakfa-logger plugin.md metadata description
fix file-logger plugin.md metadata description
fix clickhouse-logger plugin.md metadata description

Fixes #6657

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by hf400159 — APPROVED** [review]
LGTM

**Review by tzssangglass — APPROVED** [review]

**Review by hf400159 — APPROVED** [review]

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ title: skywalking-logger
 
 ## 描述
 
-`http-logger` 是一个插件，可将 Access Log 数据通过 `HTTP` 推送到 SkyWalking OAP 服务器。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，并依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/) 。
+`skywalking-logger` 是一个插件，可将 Access Log 数据通过 `HTTP` 推送到 SkyWalking OAP 服务器。如果上下文中存在 `tracing context`，插件会自动建立 `trace` 与日志的关联，并依赖于 [SkyWalking Cross Process Propagation Headers Protocol](https://skywalking.apache.org/docs/main/latest/en/protocols/skywalking-cross-process-propagation-headers-protocol-v3/) 。
 
 这将提供将 Access Log 数据作为JSON对象发送到 SkyWalking OAP 服务器的功能。
```
Comment (zaunist) [inline docs/zh/latest/plugins/skywalking-logger.md:28]:
```suggestion
这将提供将 Access Log 数据作为 JSON 对象发送到 SkyWalking OAP 服务器的功能。
```

Comment (liangliang4ward) [reply docs/zh/latest/plugins/skywalking-logger.md]: done

File: docs/zh/latest/plugins/clickhouse-logger.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ hello, world
 
 | 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
 | ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md)或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 http-logger 的 Route 或 Service 生效。 |
+| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md)或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 clickhouse-logger 的 Route 或 Service 生效。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/clickhouse-logger.md:84]:
```suggestion
| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md)或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。请注意，**该设置是全局生效的**，因此在指定 log_format 后，将对所有绑定 clickhouse-logger 的 Route 或 Service 生效。 |
```

File: docs/zh/latest/plugins/file-logger.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ hello, world
 
 | 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
 | ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 http-logger 的 Route 或 Service 生效。 |
+| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 file-logger 的 Route 或 Service 生效。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/file-logger.md:73]: ditto

File: docs/zh/latest/plugins/kafka-logger.md [hunk 4]
```diff
@@ -168,7 +168,7 @@ hello, world
 
 | 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
 | ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 http-logger 的 Route 或 Service 生效。 |
+| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 kafka-logger 的 Route 或 Service 生效。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/kafka-logger.md:171]: ditto

File: docs/zh/latest/plugins/rocketmq-logger.md [hunk 5]
```diff
@@ -165,7 +165,7 @@ hello, world
 
 | 名称             | 类型    | 必选项 | 默认值        | 有效值  | 描述                                             |
 | ---------------- | ------- | ------ | ------------- | ------- | ------------------------------------------------ |
-| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 http-logger 的 Route 或 Service 生效。 |
+| log_format       | object  | 可选   | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} |         | 以 JSON 格式的键值对来声明日志格式。对于值部分，仅支持字符串。如果是以 `$` 开头，则表明是要获取 [APISIX 变量](../../../en/latest/apisix-variable.md) 或 [Nginx 内置变量](http://nginx.org/en/docs/varindex.html)。特别的，**该设置是全局生效的**，意味着指定 log_format 后，将对所有绑定 rocketmq-logger 的 Route 或 Service 生效。 |
```
Comment (hf400159) [inline docs/zh/latest/plugins/rocketmq-logger.md:168]: ditto

File: docs/zh/latest/plugins/skywalking-logger.md [hunk 6]
```diff
@@ -87,7 +87,7 @@ hello, world
 
 ## 禁用插件
 
-在插件配置中删除相应的 json 配置以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX：
+在插件配置中删除相应的 json 配置以禁用 skywalking-logger。APISIX 插件是热重载的，因此无需重新启动 APISIX ：
```
Comment (hf400159) [inline docs/zh/latest/plugins/skywalking-logger.md:90]:
```suggestion
在插件配置中删除相应的 JSON 配置就可以禁用 `skywalking-logger` 插件。由于APISIX 插件支持热加载，因此无需重启 APISIX：
```

File: docs/zh/latest/plugins/clickhouse-logger.md (hunk 2 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/clickhouse-logger.md]: done

File: docs/zh/latest/plugins/kafka-logger.md (hunk 4 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/kafka-logger.md]: done

File: docs/zh/latest/plugins/skywalking-logger.md (hunk 6 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/skywalking-logger.md:90]: done

File: docs/zh/latest/plugins/file-logger.md (hunk 3 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/file-logger.md]: done

File: docs/zh/latest/plugins/rocketmq-logger.md (hunk 5 above)
Comment (liangliang4ward) [reply docs/zh/latest/plugins/rocketmq-logger.md]: done

**Review by hf400159 — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Review by tzssangglass — APPROVED** [review]

**Pull request merged**